### PR TITLE
Global use of must_xxx is deprecated

### PR DESCRIPTION
**Pull request by tashirosota**
[DEPRECATED: global use of must_xxx](https://github.com/brainspec/enumerize/issues/374)
Change `hoge.must_be page` to `expect(hoge).must_be page`

### CODE REVIEW

File: lib/enumerize/activerecord.rb [hunk 1]
```diff
@@ -65,7 +65,7 @@ def reload(options = nil)
         reloaded.class.enumerized_attributes.each do |attr|
           begin
             # Checks first if the enumerized attribute is in ActiveRecord::Store
-            store_attr, _ = reloaded.class.stored_attributes.detect do |store_attr, keys|
+            store_attr, _ = reloaded.class.stored_attributes.detect do |_store_attr, keys|
```
Comment (tashirosota) [inline lib/enumerize/activerecord.rb:68]: warning: shadowing outer local variable - store_attr

**Comment by tashirosota** [comment]
@nashby 
I fix `global use of must_xxx`.
Review please🙇‍♂️

**Comment by nashby** [comment]
@tashirosota hey! Thanks for this! Could you please squash these commits into one?

**Comment by tashirosota** [comment]
@nashby 
Thanks for review！
I squashed these commits into one！

**Comment by nashby** [comment]
@tashirosota thank you!

**Pull request merged**